var/spack/repos/builtin/packages/py-pyprecice/package.py

```diff
@@ -19,7 +19,9 @@ class PyPyprecice(PythonPackage):
 
     # Always prefer final version of release candidate
     version("develop", branch="develop")
-    version("2.4.0.0", sha256="8877da85df97d66966892440c7e777b667d8e6a33747304ed3762f4c33b729ec")
+    version("2.5.0.1", sha256="e2602f828d4f907ea93e34f7d4adb8db086044a75a446592a4099423d56ed62c")
+    version("2.5.0.0", sha256="9f55a22594bb602cde8a5987217728569f16d9576ea53ed00497e9046a2e1794")
+    version("2.4.0.0", sha256="e80d16417b8ce1fdac80c988cb18ae1e16f785c5eb1035934d8b37ac18945242")
```

Comment:
Any idea what the reason for the checksum change is?

Reply:
The change in the checksum was discussed before in https://github.com/spack/spack/pull/24264 and https://github.com/precice/python-bindings/issues/109. In short using the `versioneer` package to create the version is problematic for the `tar.gz` files as it stores references to branches. The solution which was decided from our side was to push an empty commit to master after releasing so that the version only points to a tag on GitHub and not to a branch. I simply forgot to push an empty commit after releasing `v2.4.0.0` and thus the checksum changed.